examples/platform/silabs/SiWx917/SiWx917/hal/rsi_hal_mcu_timer.c

```diff
@@ -257,23 +193,20 @@ uint32_t GetTickCount(void)
  * @param[out]   none
  * @return       none
  * @description  This HAL API should contain the code to create delay in milli seconds
+ *
  */
+
 void rsi_delay_ms(uint32_t delay_ms)
 {
-#ifndef RSI_WITH_OS
     uint32_t start;
-#endif
-    if (delay_ms == DELAY0)
-        return;
 
-#ifdef RSI_WITH_OS
-    vTaskDelay(delay_ms);
-#else
+    if (delay_ms == 0)
+        return;
     start = rsi_hal_gettickcount();
     do
     {
     } while (rsi_hal_gettickcount() - start < delay_ms);
-#endif
+    return;
```

Comment:
This is already the end of your function which is a void return function. No need for that return;

Reply:
Change added in the latest commit